apps/catalogue/src/views/HarmonizationView.vue

```diff
@@ -56,47 +55,43 @@
 </template>
 
 <script>
-import { mapGetters, mapActions } from "vuex";
+import { mapGetters, mapActions, mapState } from "vuex";
 import HarmonizationRow from "./HarmonizationRow.vue";
 import { Spinner } from "molgenis-components";
 
-const INITIAL_PAGE_SIZE = 10;
+const INITIAL_PAGE_SIZE = 100;
```

Comment:
harmonazation is slow , keep 10

Reply:
no, it is totally confusing now with these 10 and then next while you cannot load beyond the 100. These tabs should be more linked and are different views of the same for the users. Therefore now aligned